src/inet/BUILD.gn

```diff
@@ -65,6 +65,28 @@ source_set("inet_config_header") {
   ]
 }
 
+if (chip_device_platform == "mbed") {
+  source_set("mbed_bsd_socket") {
```

Comment:
Can you move this to a new file `src/platform/mbed/BUILD.gn`

In fact you should be able to replace this conditional with

  `assert(chip_device_platform == "mbed")`

at the top of `src/platform/mbed/BUILD.gn` if you do that.

Reply:
Done